packages/zoe/src/contractSupport/priceAuthorityTransform.js

```diff
@@ -247,5 +248,6 @@ export const makePriceAuthorityTransform = async ({
     mutableQuoteWhenGTE: makeMutableQuote('mutableQuoteWhenGTE'),
   });
 
+  // @ts-expect-error XXX promise wrappers around getQuoteIssuer
```

Comment:
hm... you prefer this carte-blanch cast to the `/** @param {PriceAuthority} pa */` annotation?

Reply:
I do but I can see (now) why it's not obviously better so if you want the change reverted I'll remove it.

We shouldn't have to infer that the output of `makePriceAuthorityTransform` is actually a PriceAuthority. The hard cast is regrettable but at least contained. The function's job is to make a PriceAuthority. Callers should be able to trust that. The hard cast is tech debt but at least when it's resolved it will be isolated. And until it is there's less cost to the calling code.

Reply:
I found fdc85824b , which localizes without a `ts-expect-error`.